5-0-functional-programming/src/test/java/com/bobocode/lambda_math_functions/FunctionsTest.java

```diff
@@ -0,0 +1,87 @@
+package com.bobocode.lambda_math_functions;
+
```

Comment:
When I wrote about redundant imports I meant that we don't need imports of old assertions (`assertEquals`, `assertThrows`). But we can't perform this class without others necessary imports.